src/foam/u2/crunch/lab/CrunchLab.js

```diff
@@ -82,32 +103,80 @@ foam.CLASS({
         };
       },
     },
+    {
+      class: 'Reference',
+      name: 'rootCapability',
+      of: 'foam.nanos.crunch.Capability',
+      help: `Root capability reference used to populate graph.
+          Graph renders prerequisites downward of the selected capabilty.`,
+      view: function(_, X) {
+        return {
+          class: 'foam.u2.view.RichChoiceView',
+          search: true,
+          allowClearingSelection: true,
+          sections: [
+            {
+              heading: 'Capabilities',
+              dao$: X.data.slot(function(showAllCapabilities, associatedUser, crunchUser) {
+                if ( crunchUser == 0 || showAllCapabilities) return this.capabilityDAO;
+                let predicate = associatedUser ?
+                    this.AND(
+                      this.EQ(this.AgentCapabilityJunction.SOURCE_ID, crunchUser),
+                      this.EQ(this.AgentCapabilityJunction.EFFECTIVE_USER, associatedUser)
+                    ) :
```

Comment:
```suggestion
                    this.OR(
                      this.AND(
                        this.EQ(this.AgentCapabilityJunction.SOURCE_ID, crunchUser),
                        this.EQ(this.AgentCapabilityJunction.EFFECTIVE_USER, associatedUser)
                      ),
                      this.EQ(this.UserCapabilityJunction.SOURCE_ID,associatedUser)
                    ) :
```
```suggestion
                    this.AND(
                      this.EQ(this.AgentCapabilityJunction.SOURCE_ID, crunchUser),
                      this.EQ(this.AgentCapabilityJunction.EFFECTIVE_USER, associatedUser)
                    ) :
```

This will allow better visibility of capabilities owned by the associated user

Reply:
actually ignore this commit suggestion - I have a better idea

Reply:
This conversation was resolved on another communication channel. 

The suggested predicate above provides all combinations of capabilities that relate to the users chosen. Which includes more capabilities in the root capability list than desired. Suggestion was dismissed.